packages/framer-motion/src/projection/node/create-projection-node.ts

```diff
@@ -1225,6 +1226,12 @@ export function createProjectionNode<I>({
              */
             copyBoxInto(this.layoutCorrected, this.layout.layoutBox)
 
+            /**
+             * Record previous tree scales before updating.
```

Comment:
Moving this before updating treeScale as apparently this would have been completely useless before - potentially leading to skipped renders (which would be hard to capture in a test)